noty-android/app/composeapp/src/main/java/dev/shreyaspatil/noty/composeapp/view/screen/SignUpScreen.kt

```diff
@@ -161,17 +163,17 @@ fun SignUpScreen(
                     backgroundColor = MaterialTheme.colors.background,
                     value = confirmPassword.value,
                     onValueChange = {
-                        isConfirmPasswordValid.value = it.text == password.value.text
+                        isConfirmPasswordValid = it.text == password.value.text
```

Comment:
@bhanup212 Here we can check it by calling a method `isPasswordAndConfirmPasswordSame()` which will add some readability.

```kotlin
isConfirmPasswordValid = AuthValidator.isPasswordAndConfirmPasswordSame(password.text, it.text)

```